web/styles/subscriptions.css

```diff
@@ -1219,6 +1208,15 @@ h4.user_group_setting_subsection_title {
             display: inline !important;
         }
     }
+
+    .icon-button {
+        font-size: 1.2em;
+    }
+
+    #open_group_info_modal,
+    #open_stream_info_modal {
+        left: 3px;
```

Comment:
Adding `left` and `margin-right` properties here is not the ideal fix. The alignment is off due to different right margins for `button-group` which is 18px and `inner-box` which is 20px. Also the padding on button to open info modal and button to deactivate or archive is different, so that should also be same if we want the perfect alignment by making the above two margins equal.